gapic/src/main/com/google/gapid/MainWindow.java

```diff
@@ -271,7 +271,7 @@ private Point getDefaultInitialLocation(Point size) {
   protected MenuManager createMenuManager() {
     MenuManager manager = new MenuManager();
 
-    // Add a dummy file menu, so the UI doesn't move once the rest of the menus are created.
+    // Add a place holder file menu, so the UI doesn't move once the rest of the menus are created.
```

Comment:
"placeholder" in one word, to be homogeneous

Reply:
done.